python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -442,7 +442,7 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
                                                         forcedStr="unforced " + self.catLabel, **plotKwargs))
             if self.config.doPlotSkyObjects:
                 plotList.append(self.plotSkyObjects(skyObjCat, "skyObjects", plotInfoDict, areaDict,
-                                                    forcedStr="unforced"))
+                                                    repoInfo, forcedStr="unforced"))
```

Comment:
You can also now get rid of the unused `repoInfo`.

Reply:
Removed.